code/modules/reagents/chemistry_properties/prop_positive.dm

```diff
@@ -492,7 +492,7 @@
 		var/obj/item/alien_embryo/embryo = content
 		if(embryo && istype(embryo))
 			if(embryo.counter > 0)
-				embryo.counter = embryo.counter - potency
+				embryo.counter = embryo.counter - potency*1.5
```

Comment:
remove the 1.5 modifier and replace it with delta_time this has been caused by beagle adding "speed" to the larva gestation and then fixing APS incorrectly. label this as a fix since his PR was likely not suposed to make APS work worse 
#3844

Reply:
Will do. I presume you mean to add delta_time and not multiply by it? Or do you genuinely mean to multiply by delta_time